Whisky/Views/Updater/UpdatePreviewView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  UpdateUI.swift
+//  Whisky
+//
+//  This file is part of Whisky.
+//
+//  Whisky is free software: you can redistribute it and/or modify it under the terms
+//  of the GNU General Public License as published by the Free Software Foundation,
+//  either version 3 of the License, or (at your option) any later version.
+//
+//  Whisky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+//  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+//  See the GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License along with Whisky.
+//  If not, see https://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import Sparkle
+import MarkdownUI
+
+struct UpdatePreviewView: View {
+    let dismiss: () -> Void
+    let install: () -> Void
+    @Binding var markdownText: String
+    @Binding var nextVersion: String
```

Comment:
I don't see any reason why these need to be `@Bindng` a simple let should work just fine here.

Reply:
I tried that yet somehow it didn't work.